common/flatpak-dir.c

```diff
@@ -9677,7 +9801,19 @@ flatpak_dir_get_locale_subpaths (FlatpakDir *self)
       if (flatpak_dir_is_user (self))
         subpaths = flatpak_get_current_locale_subpaths ();
       else
-        subpaths = g_new0 (char *, 1);
+        {
+          GDBusProxy *proxy = get_accounts_dbus_proxy ();
+          if (proxy)
+            {
+              subpaths = get_locale_subpaths_from_accounts_dbus (proxy);
```

Comment:
I analyzed AccountService and found that it only monitors local users and groups by three configuration files: "/etc/passwd", "/etc/shadow" and "/etc/group". So I think we can ignore ldap user.
Local users usually don't have more than 100, so I think get user languages list by dbus api is feasible.